src/foam/nanos/boot/NSpec.js

```diff
@@ -215,6 +215,17 @@ foam.CLASS({
       transient: true,
       javaGetter: 'return getName();',
       getter: function() { return this.name; }
+    },
+    {
+      class: 'Int',
+      name: 'uidKey',
+      value: -1,
+      javaSetter: `
+        uidKey_ = val;
```

Comment:
curious why this is needed?

Reply:
The uidKey will be set in easyDAO when setting up the FUIDDAO, but the nspec instance in the easyDAO is frozen and I'm hesitant to fclone the nspec to update the uidKey then save back to the nspecDAO.

Might be a bit hacky this way, I'm open to suggestions @jlhughes @kgrgreer.

Reply:
Removed 'uidKey' and use fuid keys instead.